{"name": "test_02", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 > 2", "trace": "self = <test221010.public.test_email.TestEmail object at 0x0000017F5E211190>\n\n    def test_02(self):\n        with allure.step(\"先输出描述\"):\n            print(\"第二条用例\")\n>       assert 1>2\nE       assert 1 > 2\n\ntest_email.py:22: AssertionError"}, "steps": [{"name": "先输出描述", "status": "passed", "start": 1672803177474, "stop": 1672803177474}], "start": 1672803177474, "stop": 1672803177474, "uuid": "4f7fe78d-24c2-418c-b9f7-dfc93e98cb74", "historyId": "bfa9f6410e18e4dc946d77e960f99137", "testCaseId": "0ebd88c53b5221fffca92e879d0730dc", "fullName": "test_email.TestEmail#test_02", "labels": [{"name": "suite", "value": "test_email"}, {"name": "subSuite", "value": "TestEmail"}, {"name": "host", "value": "DESKTOP-8H78TQB"}, {"name": "thread", "value": "34124-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_email"}]}